hs_core/tests/api/rest/test_resmap.py

```diff
@@ -0,0 +1,145 @@
+import os
+import json
+import tempfile
+import shutil
+from rest_framework import status
+from rdflib import Graph, term
+
+from hs_core.hydroshare import resource
+from .base import ResMapTestCase
+
+
+class TestResourceMap(ResMapTestCase):
```

Comment:
Though I don't know much about the rdflib, it seems we can use this test pattern for testing resourcemetadata.xml. I am going to create an issue in GitHub so that we don't forget to do it.